Python/Tests/IronPython/IronPythonAnalysisTest.cs

```diff
@@ -490,24 +490,24 @@ import wpf
         public void XamlEmptyXName() {
             // [Python Tools] Adding attribute through XAML in IronPython application crashes VS.
             // http://pytools.codeplex.com/workitem/743
-            using (var analyzer = CreateAnalyzer()) {
-                string xamlPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.xaml");
-                string pyPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.py");
-                var xamlEntry = analyzer.Analyzer.AddXamlFile(xamlPath);
-                var pyEntry = analyzer.AddModule("EmptyXName", File.ReadAllText(pyPath), pyPath);
-
-                xamlEntry.ParseContent(new FileStreamReader(xamlPath), null);
-
-                using (var stream = new FileStreamReader(pyPath)) {
-                    var parser = Parser.CreateParser(stream, PythonLanguageVersion.V27, new ParserOptions() { BindReferences = true });
-                    using (var p = pyEntry.BeginParse()) {
-                        p.Tree = parser.ParseFile();
-                        p.Complete();
-                    }
-                }
-
-                pyEntry.Analyze(CancellationToken.None);
-            }
+            //using (var analyzer = CreateAnalyzer()) {
+            //    string xamlPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.xaml");
+            //    string pyPath = TestData.GetPath(@"TestData\Xaml\EmptyXName.py");
+            //    var xamlEntry = analyzer.Analyzer.AddXamlFile(xamlPath);
```

Comment:
This test was commented out because of the `AddXamlFile` method. It should be actually removed from `Analysis` with the rest of the `IronPython`-specific code, but it requires some changes in public interfaces. This doesn't block separation by itself though, cause it is called only from Analyzer.

Reply:
Looks like you could change signature to return IProjectEntry instead of IXamlProjectEntry. Then it's easier to make it public and keep the test alive. I'm just worried we'll forget about this commented out test.

Reply:
There are more changes required here. I need to remove IronPython bits from Analysis, cause they require separate compilation for .net framework. But I want it to be a separate PR.